pkg/schema/artifact.go

```diff
@@ -39,23 +39,19 @@ func (a ReleaseArtifact) FileName() string {
 
 	}
 
-	var re = regexp.MustCompile(`v(?P<major>\d+)\.(?P<minor>\d+).+`)
-	match := re.FindStringSubmatch(a.ReleaseVersion)
-	result := make(map[string]string)
-	for i, name := range re.SubexpNames() {
-		if i != 0 && name != "" {
-			result[name] = match[i]
+	if semver.Compare(a.ReleaseVersion, "v2.4.0") < 0 {
```

Comment:
YEEES

Reply:
I did not know we were doing regex for the versions xD

Reply:
yup that was a stupid decision from me from the past 🤣